spec/cluster_overlap_spec.rb

```diff
@@ -42,13 +42,6 @@
     end
   end
 
-  describe "#organization_in_cluster" do
```

Comment:
why was this test taken out? no longer relevant? should be replaced with another test?

Reply:
Duplicative with the `def organizations_in_cluster` found in lib/cluster.rb and its spec.